mscore/editstyle.ui

```diff
@@ -4331,6 +4164,366 @@
+               <normaloff>data/icons/edit-reset.svg</normaloff>data/icons/edit-reset.svg</iconset>
+             </property>
+            </widget>
+           </item>
+           <item row="4" column="0" colspan="6">
+            <widget class="QGroupBox" name="oldStyleMultiMeasureRests">
+             <property name="title">
+              <string>Old-Style Multimeasure Rests</string>
+             </property>
+             <property name="checkable">
+              <bool>true</bool>
+             </property>
+             <property name="checked">
+              <bool>false</bool>
+             </property>
+             <layout class="QGridLayout" name="gridLayout_30">
+              <item row="0" column="3">
+               <spacer name="horizontalSpacer_15">
+                <property name="orientation">
+                 <enum>Qt::Horizontal</enum>
+                </property>
+                <property name="sizeHint" stdset="0">
+                 <size>
+                  <width>40</width>
+                  <height>20</height>
+                 </size>
+                </property>
+               </spacer>
+              </item>
+              <item row="0" column="0">
+               <widget class="QLabel" name="label_146">
+                <property name="text">
+                 <string>Maximum number of measures:</string>
+                </property>
+               </widget>
+              </item>
+              <item row="0" column="2">
+               <widget class="QToolButton" name="resetMMRestOldStyleMaxMeasures">
+                <property name="text">
+                 <string>...</string>
+                </property>
+                <property name="icon">
+                 <iconset>
+                  <normaloff>data/icons/edit-reset.svg</normaloff>data/icons/edit-reset.svg</iconset>
+                </property>
+               </widget>
+              </item>
+              <item row="0" column="1">
+               <widget class="QSpinBox" name="mmRestOldStyleMaxMeasures"/>
+              </item>
+              <item row="1" column="1">
+               <widget class="QDoubleSpinBox" name="mmRestOldStyleSpacing">
+                <property name="suffix">
+                 <string>sp</string>
+                </property>
+                <property name="maximum">
+                 <double>4.000000000000000</double>
+                </property>
+                <property name="singleStep">
+                 <double>0.010000000000000</double>
+                </property>
+               </widget>
+              </item>
+              <item row="1" column="0">
+               <widget class="QLabel" name="label_147">
+                <property name="text">
+                 <string>Spacing of symbols:</string>
+                </property>
+               </widget>
+              </item>
+              <item row="1" column="2">
+               <widget class="QToolButton" name="resetMMRestOldStyleSpacing">
```

Comment:
Maybe a different step value than 0.01 (which is apparently the default) makes sense here.  Most settings with defaults in this range (somewhere close to 1 sp) have a step of more like 0.1